src/main/java/neuro/command/DeleteCommand.java

```diff
@@ -0,0 +1,49 @@
+package neuro.command;
+
+import neuro.Storage;
+import neuro.Ui;
+import neuro.task.Task;
+import neuro.task.TaskList;
+
+/**
+ * The {@code DeleteCommand} class represents a command to delete a task from the task list.
+ */
+public class DeleteCommand extends Command {
+    private final int index;
+
+    /**
+     * Constructs a DeleteCommand object with the specified index of the task to be deleted.
+     *
+     * @param index The index of the task to be deleted from the task list.
+     */
+    public DeleteCommand(int index) {
+        this.index = index;
+    }
+
+    /**
+     * Executes the delete command to delete the task at the index from the task list.
+     *
+     * @param tasks the task list on which the command operates
+     * @param ui the user interface for interacting with the user
+     * @param storage the storage for saving and loading tasks
+     */
+    @Override
```

Comment:
Great descriptions in Javadoc that provide explanations and increases readibility and understanding of code throughout!